Solutions/Microsoft 365 Defender/Workbooks/MDO-Workbook.json

```diff
@@ -0,0 +1,1317 @@
+                        {
+                          "operator": "==",
+                          "thresholdValue": "Delivered",
+                          "representation": "success",
+                          "text": "{0}{1}"
+                        },
+                        {
+                          "operator": "==",
+                          "thresholdValue": "Blocked",
+                          "representation": "failed",
+                          "text": "{0}{1}"
+                        },
+                        {
+                          "operator": "==",
+                          "thresholdValue": "Junked",
+                          "representation": "Disable",
+                          "text": "{0}{1}"
+                        },
+                        {
+                          "operator": "Default",
+                          "thresholdValue": null,
+                          "representation": "unknown",
+                          "text": "{0}{1}"
+                        }
+                      ]
+                    }
+                  },
+                  {
+                    "columnMatch": "ConfidenceLevel",
+                    "formatter": 0,
+                    "formatOptions": {
+                      "customColumnWidthSetting": "25ch"
+                    }
+                  },
+                  {
+                    "columnMatch": "AttachmentCount",
+                    "formatter": 4,
+                    "formatOptions": {
+                      "palette": "blue"
+                    }
+                  },
+                  {
+                    "columnMatch": "UrlCount",
+                    "formatter": 4,
+                    "formatOptions": {
+                      "palette": "blue"
+                    }
+                  },
+                  {
+                    "columnMatch": "NetworkMessageId",
+                    "formatter": 5
+                  }
+                ]
+              }
+            },
+            "customWidth": "35",
+            "conditionalVisibility": {
+              "parameterName": "Nav",
+              "comparison": "isEqualTo",
+              "value": "Authentication"
+            },
+            "name": "query - 6"
+          }
+        ]
+      },
+      "conditionalVisibility": {
+        "parameterName": "Nav",
+        "comparison": "isEqualTo",
+        "value": "Email"
+      },
+      "name": "EmailGroup"
+    },
+    {
+      "type": 12,
+      "content": {
+        "version": "NotebookGroup/1.0",
+        "groupType": "editable",
+        "items": [
+          {
+            "type": 3,
+            "content": {
+              "version": "KqlItem/1.0",
+              "query": "EmailEvents \r\n| project Timestamp, AR=parse_json(AuthenticationDetails) , NetworkMessageId, EmailDirection, SenderFromAddress, ThreatTypes, DetectionMethods   \r\n| evaluate bag_unpack(AR) \r\n| where DMARC == \"fail\" \r\n| summarize count() by bin(Timestamp, 1d) ",
```

Comment:
Change the comparison to =~
Applicable to all the queries in this section.

Reply:
done